vllm/config/__init__.py

```diff
@@ -8,17 +8,20 @@
 import hashlib
 import inspect
 import json
+import os
 import textwrap
 import uuid
 import warnings
-from collections.abc import Mapping
+from collections.abc import Iterable, Mapping
 from contextlib import contextmanager
 from dataclasses import MISSING, Field, field, fields, is_dataclass, replace
 from functools import cached_property, lru_cache
 from importlib.util import find_spec
+from pathlib import Path
 from typing import (TYPE_CHECKING, Any, Callable, ClassVar, Literal, Optional,
                     Protocol, TypeVar, Union, cast, get_args)
 
+import model_signing
```

Comment:
These imports can be removed because they're not only used in `signature_verification.py`

Reply:
Relying too much on pre-commit. It doesn't remove them. Thanks.